SQL/New_patches/2020-08-12_add_CNV_CPG_data_for_genomic_browser.sql

```diff
@@ -0,0 +1,25 @@
+--Add CNV data for some candidates
```

Comment:
files in this directory get run by real projects with real data. this patch is adding mock data useful for testing.

This addition while very very helpful should not be in the patch directory. it's sufficient to make the changes to the RB files only.

if it makes it easier to understand:
 - anything that should be ran on all projects should also be ran on RB
 - however patches created FOR RB should never be ran on projects

Reply:
Okay this makes sense. I'll remove the patch and leave the changes in the Raisinbread files.

Reply:
Alex can you also add this note to the Cheat sheet?  this isn't isn't available to the team unless rida tells them otherwise.